util/security/jwtutil.py

```diff
@@ -90,7 +88,7 @@ def _reject_future_iat(self, payload, now, leeway):
             raise InvalidIssuedAtError("Issued At claim (iat) cannot be in" " the future.")
 
 
-def decode(jwt, key="", verify=True, algorithms=None, options=None, **kwargs):
+def decode(jwt, key="", verify=True, algorithms=None, options={}, **kwargs):
```

Comment:
using an empty dict as default argument is not a good idea, we better stick with `options=None`, then in the function body assign it to an empty dict when `options is None`.

from the docs[1]:
> *Important warning:* The default value is evaluated only once. This makes a difference when the default is a mutable object such as a list, dictionary, or instances of most classes.

[1] https://docs.python.org/3/tutorial/controlflow.html#default-argument-values